src/test/java/com/gdschongik/gdsc/domain/member/domain/MemberTest.java

```diff
@@ -41,14 +41,100 @@ class 게스트_회원가입시 {
         }
     }
 
+    @Nested
+    class 준회원_승급_만족여부 {
+        @Test
+        void 기본_회원정보_기입하지_않았으면_isAllVerified는_false이다() {
+            // given
+            Member member = Member.createGuestMember(OAUTH_ID);
+
+            member.verifyDiscord(DISCORD_USERNAME, NICKNAME);
+            member.completeUnivEmailVerification(UNIV_EMAIL);
+            member.verifyBevy();
+
+            // when & then
+            assertThat(member.isAllVerified()).isFalse();
+        }
+
+        @Test
+        void 재학생_인증하지_않았으면_isAllVerified는_false이다() {
+            // given
+            Member member = Member.createGuestMember(OAUTH_ID);
+
+            member.verifyDiscord(DISCORD_USERNAME, NICKNAME);
+            member.completeUnivEmailVerification(UNIV_EMAIL);
+            member.verifyBevy();
+
+            // when & then
+            assertThat(member.isAllVerified()).isFalse();
+        }
+
+        @Test
+        void 디스코드_인증하지_않았으면_isAllVerified는_false이다() {
+            // given
+            Member member = Member.createGuestMember(OAUTH_ID);
+
+            member.updateBasicMemberInfo(STUDENT_ID, NAME, PHONE_NUMBER, D022, EMAIL);
+            member.completeUnivEmailVerification(UNIV_EMAIL);
+            member.verifyBevy();
+
+            // when & then
+            assertThat(member.isAllVerified()).isFalse();
+        }
+
+        @Test
+        void Bevy_연동하지_않았으면_isAllVerified는_false이다() {
+            // given
+            Member member = Member.createGuestMember(OAUTH_ID);
+
+            member.updateBasicMemberInfo(STUDENT_ID, NAME, PHONE_NUMBER, D022, EMAIL);
+            member.completeUnivEmailVerification(UNIV_EMAIL);
+            member.verifyDiscord(DISCORD_USERNAME, NICKNAME);
+
+            // when & then
+            assertThat(member.isAllVerified()).isFalse();
+        }
+
+        @Test
+        void 기본_회원정보_작성_디스코드인증_Bevy인증_재학생인증하면_isAllVerified는_true이다() {
```

Comment:
```suggestion
        void 준회원_가입조건을_모두_충족했다면_isAllVerified는_true이다() {
```

이렇게 하면 나중에 준회원 가입조건에 뭔가 추가되거나 제외되더라도,
테스트 이름을 바꾸지 않아도 될 것 같습니다!

Reply:
이름에 대해 고민하고 있었는데 이게 좋은것같네용 감삼당